client/src/pages/Availability/Availability.tsx

```diff
@@ -0,0 +1,50 @@
+import { useState, SyntheticEvent } from 'react';
+import { Paper, Typography, Box, Button, List } from '@material-ui/core';
+import useStyles from './useStyles';
+import { useAuth } from '../../context/useAuthContext';
+import { useHistory, useParams } from 'react-router-dom';
+import { useSnackBar } from '../../context/useSnackbarContext';
+import DayAvailability from '../../components/DayAvailability/DayAvailability';
+import { updateAvailability } from '../../helpers/APICalls/profile';
+export default function Availability(): JSX.Element {
+  const classes = useStyles();
+  const daysOfWeek = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'];
+  const { updateSnackBarMessage } = useSnackBar();
+  const history = useHistory();
+  interface Slot {
```

Comment:
Please define this interface outside of the component

Reply:
fixed